tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -429,6 +431,20 @@ public:
    }
 
    ////////////////////////////////////////////////////////////////////////////
+   /// \brief Allow to refer to a column with a different name
+   /// \param[in] name of the column to be aliased
+   /// \param[in] new name for the column
```

Comment:
`param[in]`s do not match the parameter names

Reply:
ops. will fix.